spec/controllers/admins/users_controller_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Api::V1::Admin::UsersController, type: :controller do
+  before do
+    request.headers['ACCEPT'] = 'application/json'
+  end
+
+  describe '#active_users' do
+    it 'ids of rooms in response are matching room ids that belong to current_user' do
+      # TODO: Change this test to create active users and not just any users
+      users = create_list(:user, 5)
+      get :active_index
+      expect(response).to have_http_status(:ok)
+      response_user_ids = JSON.parse(response.body)['data'].map { |user| user['id'] }
+      expect(response_user_ids).to eq(users.pluck(:id))
```

Comment:
This is a flaky test, the order of ids may differ braking the tests run.
I would do this instead:
```
expect(JSON.parse(response.body)['data'].pluck('id')).to match(users.pluck(:id))
```

Reply:
Order doesn't seem to be considered when using `eq`, we have many tests that follow this same format and never fail

Reply:
Those tests were fixed by Samuel since they were occasionally failing. We should make the change to `match_array`